Assets/Scripts/Input/InputSystem.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using UnityEngine.AddressableAssets;
+using UnityEngine.InputSystem;
+
+namespace Input 
+{
+    public static class InputSystem
+    {
+        private static InputActions _inputAction = new();
+        private static Dictionary<System.Action<InputAction.CallbackContext>, InputAction> _subscribedInputs = new();
+
+        public static InputAction TryGetAction(string actionName)
+        {
+            InputAction action = _inputAction.FindAction(actionName);
+            
+            return action;
+        }
```

Comment:
Can also just be a one-liner

```suggestion
        public static InputAction TryGetAction(string actionName) => _inputAction.FindAction(actionName);
```

Reply:
Did that